### ISSUE TITLE
A user comment

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#1210]
I have just seen this comment from a user of the WCAG (someone at Expedia actually): 

> Many companies have communicated a commitment to accessibility, but WCAG can be convoluted and challenging to digest.

I suggest that a lot of this "convoluted and challenging to digest" issue is due to not enough attention being paid to using simple wording, and not saying what we actually mean in plain English. This is still happening in WCAG 2.2, and I just want to highlight this person's comment in order to draw everyone's attention to the issue.

**Comment by detlevhfischer** [comment]
@guyhickling Throughout work on SCs thare is the constant trade-off between 'keeping it simple' and 'phrasing it such that it is unambiguous in terms of what exactly is required / tests in a PASS/FAIL conformance test. The issue is being addressed in WCAG 3.0 Silver, but I believe the general tension between ease of understanding and precision of expression is likely to remain.

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do that and convert it to a Discussion.
